src/js/classes/Table.js

```diff
@@ -307,6 +311,36 @@ export class RankedTable {
     }
   }
 
+  initRender() {
```

Comment:
Is there a better name for this? Perhaps something that follows convention?

Reply:
Maybe just `init`? I don't have strong feelings.